client/src/components/Login/Login.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import {
+  Form, Input, Button,
+} from 'antd';
+import axios from 'axios';
+import { useDispatch } from 'react-redux';
+import './style.css';
+import { setUser } from '../../redux/features/auth/user';
+
+function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+  const dispatch = useDispatch();
+  const { Item } = Form;
+  const { Password } = Input;
+  const changeEmail = (e) => {
+    e.preventDefault();
+    setEmail(e.target.value);
+  };
+
+  const changePassword = (e) => {
+    e.preventDefault();
+    setPassword(e.target.value);
+  };
+
+  const onFinish = async () => {
+    try {
+      const user = await axios.post('/api/login', { email, password });
+      dispatch(setUser(user.data.data.user));
+      setError('');
+    } catch (err) {
+      setError(err.response.data.message);
```

Comment:
Check the message antd component and use it, that you will not need the error state